File: fixtures/r/post_quotes_inside_data.r
require(httr)

headers = c(
  `Content-Type` = 'application/x-www-form-urlencoded'
)

data = list(
  `field` = 'don\'t you like quotes'
)

res <- httr::POST(url = 'http://google.com', httr::add_headers(.headers=headers), body = data)

File: test/fixtures/ruby-httparty/post_quotes_inside_data.rb
require 'httparty'

url = 'http://localhost:28139'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = {
  'field' => "don't you like quotes"
}
res = HTTParty.post(url, headers: headers, body: body)
